File: Sorting 1/bubbleSort.js
const bubble=(arr)=>{
    let n=arr.length;
    for(let i=n-1;i>=1;i++){
        let swapped=false; 
        for(let j=0;j<=i-1;j++){
            if(arr[j]>arr[j+1]){
                [arr[j],arr[j+1]]=[arr[j+1],arr[j]];
                swapped=true;
            }
        }
        if(!swapped){
            break;
        }
    }
}
let arr=[13, 46, 24, 52, 20, 9];
bubble(arr);
console.log(arr);

File: Sorting 1/Bubble_Sort/bubbleSort.py
def bubble(arr):
    n=len(arr)
    for i in range(n-1,0,-1):
        didSwap=0
        for j in range(0,i):
            if arr[j]>arr[j+1]:
                [arr[j],arr[j+1]]=[arr[j+1],arr[j]]
                didSwap=1
        
        if didSwap==0:
            break

arr=[14, 46, 24, 52, 20, 9]
bubble(arr)
print(arr)